src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,57 @@
+package seedu.duke;
+
+import seedu.duke.exception.DukeIoException;
+import seedu.duke.task.Task;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Storage class that handles saving and loading TaskList data as text file.
+ */
+public class Storage {
+
+    private final Path filePath;
+
+    Storage(Path filePath) {
+        this.filePath = filePath;
+    }
+
+    List<String> load() throws DukeIoException {
+        if (!Files.exists(filePath)) {
+            throw new DukeIoException("Path does not exists.");
+        }
+
+        try {
+            return Files.readAllLines(filePath);
+        } catch (IOException e) {
+            throw new DukeIoException(e.getMessage());
+        }
+    }
+
+    /**
+     * Save given TaskList object as text file.
```

Comment:
Try to use present tense such as saves, gets, puts, displays, calculates etc. for all descriptions in JavaDoc.